iOS/Features/Spot/Sources/SpotView/SpotViewController.swift

```diff
@@ -0,0 +1,223 @@
+            self.subTextLabel.heightAnchor.constraint(equalToConstant: Metrix.SubTextLabel.height),
+            
+            self.textLabel.topAnchor.constraint(equalTo: self.imageView.bottomAnchor, constant: Metrix.TextLabel.topInset),
+            self.subTextLabel.topAnchor.constraint(equalTo: self.textLabel.bottomAnchor, constant: Metrix.SubTextLabel.topInset)
+        ])
+        self.subTextLabel.textAlignment = .center
+    }
+    
+    private func configureButtonsLayout() {
+        let buttons: [MSRectButton] = [self.cancelButton, self.completeButton]
+        buttons.forEach { button in
+            self.view.addSubview(button)
+            button.translatesAutoresizingMaskIntoConstraints = false
+            NSLayoutConstraint.activate([
+                button.heightAnchor.constraint(equalToConstant: Metrix.Button.height),
+                button.widthAnchor.constraint(equalToConstant: Metrix.Button.width),
+                button.bottomAnchor.constraint(equalTo: self.view.bottomAnchor, constant: -Metrix.Button.bottomInset)
+            ])
+        }
+        NSLayoutConstraint.activate([
+            self.cancelButton.trailingAnchor.constraint(equalTo: self.view.centerXAnchor, constant: -Metrix.Button.insetFromCenterX),
+            self.completeButton.leadingAnchor.constraint(equalTo: self.view.centerXAnchor, constant: Metrix.Button.insetFromCenterX),
+        ])
+    }
+    
+    // MARK: - UI Components: Style
+    
+    private func configureStyle() {
+        MSFont.registerFonts()
+        self.view.backgroundColor = .msColor(.primaryBackground)
+        self.configureImageViewStyle()
+        self.configureLabelsStyle()
+        self.configureButtonsStyle()
+    }
+    
+    private func configureImageViewStyle() {
+        self.imageView.contentMode = .scaleAspectFill
+    }
+    
+    private func configureLabelsStyle() {
+        self.textLabel.font = .msFont(.subtitle)
+        self.subTextLabel.font = .msFont(.caption)
+        self.subTextLabel.textColor = .msColor(.secondaryTypo)
+    }
+    
+    private func configureButtonsStyle() {
+        self.cancelButton.image = .msIcon(.close)
+        self.completeButton.image = .msIcon(.check)
+    }
+    
+    // MARK: - Configure: Action
+    
+    private func configureAction() {
+        self.configureCancelAction()
+        self.configureCompleteAction()
+    }
+    
+    private func configureCancelAction() {
+        self.cancelButton.addTarget(self, action: #selector(cancelButtonTapped), for: .touchUpInside)
+    }
+    
+    private func configureCompleteAction() {
+        self.completeButton.addTarget(self, action: #selector(completeButtonTapped), for: .touchUpInside)
+    }
+    
+    // MARK: - Configure: Value
+    
+    private func configureValue() {
+        self.configureImageViewValue()
+        self.configureLabelsValue()
+    }
+    
+    private func configureImageViewValue() {
+        self.imageView = UIImageView(image: self.image)
```

Comment:
```suggestion
        self.imageView.image = self.image
```

`UIImageView` 자체가 아니라 image만 교체해줘도 될 것 같습니다!